etc/pfmb/hosts.conf

```diff
@@ -1,2 +1,2 @@
-pfsense.example.com
-10.10.1.1:80
+pfsense.example.com 8080 PfsenseExemple
+10.10.1.1 80 Test1
```

Comment:
These should be separated with `:` instead of ` ` (space) since the IFS is using `:` to separate them.